ci/build_container/build_recipes/luajit.sh

```diff
@@ -59,12 +57,14 @@ index f7f81a4..e698517 100644
  ##############################################################################
 EOF
 
+cd LuaJIT-"$VERSION"
+
 if [[ "${OS}" == "Windows_NT" ]]; then
   cd src
   ./msvcbuild.bat debug
 
-  mkdir -p "$THIRDPARTY_BUILD/include/luajit-2.0"
-  cp *.h* "$THIRDPARTY_BUILD/include/luajit-2.0"
+  mkdir -p "$THIRDPARTY_BUILD/include/luajit-2.1"
+  cp *.h* "$THIRDPARTY_BUILD/include/luajit-2.1"
```

Comment:
Thinking about it some more, it seems that this version upgrade is especially painful because the version is baked into these paths. It doesn't help this time around, but if we removed the -2.0/-2.1 suffix here, doesn't that make it simpler to upgrade in the future?

@mattklein123 wdyt?

Reply:
I think I tried to do that, but had some problems with old style cast warning (see below in the diff). If there is a way to strip this that would be great, but also, realistically this is probably the last version of LuaJIT that will ever be released (it's a dead project for all intents and purposes) so it probably doesn't matter that much?